slamd/discovery/processing/strategies/csv_strategy.py

```diff
@@ -27,10 +32,17 @@ def create_dataset(cls, file_data):
             raise ValueNotSupportedException('You cannot use the name temporary for your dataset!')
 
         try:
-            dataset = Dataset(name=file_name, dataframe=read_csv(file_data, delimiter=delimiter, decimal=decimal))
+            dataset = Dataset(
+                name=file_name,
+                dataframe=read_csv(file_data, delimiter=delimiter, decimal=decimal, on_bad_lines='error')
+            )
         except:
             raise ValueNotSupportedException('The dataset you submitted could not be read.')
 
+        for col in dataset.dataframe.columns:
+            # errors='ignore' => If non-numeric columns can not be converted, they are returned without conversion
+            dataset.dataframe[col] = pd.to_numeric(dataset.dataframe[col], errors='ignore')
+
```

Comment:
maybe we should do the conversion only for cols which are not already numeric. Currently we do it for any column. As usually the cols are already numeric, most of the iterations in the loop are actually not needed.

Reply:
with the last commit, the  loop makes sense after all, as we need to trim whitespaces for all datarows in all cols anyway

Reply:
discussed details about the pr -> so we can resolve these points